Dockerfile

```diff
@@ -1,3 +1,12 @@
-# This Dockerfile only exists to trigger Dependabot to update this tooling
-# dependency.
-FROM namely/protoc-all@sha256:07f1ba9dbe11f5675e2efc8617c9552217dc4c3eb5ccd108f7c3889878dbae50 # v1.50_0
+FROM namely/protoc-all@sha256:07f1ba9dbe11f5675e2efc8617c9552217dc4c3eb5ccd108f7c3889878dbae50
+
+# Install Python pip.
+RUN set -ex && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+    python3-pip
```

Comment:
Because we're executing the plugin within Docker, I need to install the `betterproto` in the image. For that, I need `pip`.

Therefore, I've had to begin building a new image on top of `namely/protoc-all` since the base image no longer has everything we need. This is covered by the Makefile.